-1 
Tested at: 0d80b3c6419a486e8d44751fe1db983a3c9aa86c
 When I ran the RelVals I found an error in the following worklfows: 
 4.53 step2
<pre>runTheMatrix-results/4.53_RunPhoton2012B+RunPhoton2012B+HLTD+RECODreHLT+HARVESTDreHLT+MINIAODDreHLT/step2_RunPhoton2012B+RunPhoton2012B+HLTD+RECODreHLT+HARVESTDreHLT+MINIAODDreHLT.log

----- Begin Fatal Exception 18-May-2015 14:29:31 UTC-----------------------

An exception of category 'FallbackFileOpenError' occurred while

   [0] Constructing the EventProcessor

   [1] Constructing input source of type PoolSource

   [2] Calling RootInputFileSequence::initFile()

   [3] Calling StorageFactory::open()

   [4] Calling XrdFile::open()

Exception Message:

Failed to open the file 'root://xrootd-cms.infn.it//store/data/Run2012B/SinglePhoton/RAW/v1/000/194/533/1084D9DA-AEA2-E111-B3AB-001D09F24DA8.root'

   Additional Info:

      [a] XrdCl::File::Open(name='root://eoscms//eos/cms/store/data/Run2012B/SinglePhoton/RAW/v1/000/194/533/1084D9DA-AEA2-E111-B3AB-001D09F24DA8.root?svcClass=default', flags=0x10, permissions=0660) => error '[ERROR] Server responded with an error: [3010] Unable to give access - user access restricted - not authorized identity used ; Permission denied

' (errno=3010, code=400). No additional data servers were found.

      [b] Last URL tried: root://eoscms:1094//eos/cms/store/data/Run2012B/SinglePhoton/RAW/v1/000/194/533/1084D9DA-AEA2-E111-B3AB-001D09F24DA8.root?svcClass=default&tried=

      [c] Problematic data server: eoscms:1094

      [d] Disabled source: eoscms:1094

      [e] Input file root://eoscms//eos/cms/store/data/Run2012B/SinglePhoton/RAW/v1/000/194/533/1084D9DA-AEA2-E111-B3AB-001D09F24DA8.root?svcClass=default could not be opened.

Fallback Input file root://xrootd-cms.infn.it//store/data/Run2012B/SinglePhoton/RAW/v1/000/194/533/1084D9DA-AEA2-E111-B3AB-001D09F24DA8.root also could not be opened.

Original exception info is above; fallback exception info is below.

      [f] XrdCl::File::Open(name='root://xrootd-cms.infn.it//store/data/Run2012B/SinglePhoton/RAW/v1/000/194/533/1084D9DA-AEA2-E111-B3AB-001D09F24DA8.root', flags=0x10, permissions=0660) => error '[ERROR] Server responded with an error: [3010] Unable to give access - user access restricted - not authorized identity used ; Permission denied

' (errno=3010, code=400). No additional data servers were found.

      [g] Last URL tried: root://eoscms.cern.ch:1094//store/data/Run2012B/SinglePhoton/RAW/v1/000/194/533/1084D9DA-AEA2-E111-B3AB-001D09F24DA8.root?tried=

      [h] Problematic data server: eoscms.cern.ch:1094

      [i] Disabled source: eoscms.cern.ch:1094

----- End Fatal Exception -------------------------------------------------

</pre>
4.22 step2
<pre>runTheMatrix-results/4.22_RunCosmics2011A+RunCosmics2011A+RECOCOSD+ALCACOSD+SKIMCOSD+HARVESTDC/step2_RunCosmics2011A+RunCosmics2011A+RECOCOSD+ALCACOSD+SKIMCOSD+HARVESTDC.log

----- Begin Fatal Exception 18-May-2015 14:30:07 UTC-----------------------

An exception of category 'FallbackFileOpenError' occurred while

   [0] Constructing the EventProcessor

   [1] Constructing input source of type PoolSource

   [2] Calling RootInputFileSequence::initFile()

   [3] Calling StorageFactory::open()

   [4] Calling XrdFile::open()

Exception Message:

Failed to open the file 'root://xrootd-cms.infn.it//store/data/Run2011A/Cosmics/RAW/v1/000/160/960/049F6443-8E53-E011-A943-003048F117EA.root'

   Additional Info:

      [a] XrdCl::File::Open(name='root://eoscms//eos/cms/store/data/Run2011A/Cosmics/RAW/v1/000/160/960/049F6443-8E53-E011-A943-003048F117EA.root?svcClass=default', flags=0x10, permissions=0660) => error '[ERROR] Server responded with an error: [3010] Unable to give access - user access restricted - not authorized identity used ; Permission denied

' (errno=3010, code=400). No additional data servers were found.

      [b] Last URL tried: root://eoscms:1094//eos/cms/store/data/Run2011A/Cosmics/RAW/v1/000/160/960/049F6443-8E53-E011-A943-003048F117EA.root?svcClass=default&tried=

      [c] Problematic data server: eoscms:1094

      [d] Disabled source: eoscms:1094

      [e] Input file root://eoscms//eos/cms/store/data/Run2011A/Cosmics/RAW/v1/000/160/960/049F6443-8E53-E011-A943-003048F117EA.root?svcClass=default could not be opened.

Fallback Input file root://xrootd-cms.infn.it//store/data/Run2011A/Cosmics/RAW/v1/000/160/960/049F6443-8E53-E011-A943-003048F117EA.root also could not be opened.

Original exception info is above; fallback exception info is below.

      [f] XrdCl::File::Open(name='root://xrootd-cms.infn.it//store/data/Run2011A/Cosmics/RAW/v1/000/160/960/049F6443-8E53-E011-A943-003048F117EA.root', flags=0x10, permissions=0660) => error '[ERROR] Server responded with an error: [3010] Unable to give access - user access restricted - not authorized identity used ; Permission denied

' (errno=3010, code=400). No additional data servers were found.

      [g] Last URL tried: root://eoscms.cern.ch:1094//store/data/Run2011A/Cosmics/RAW/v1/000/160/960/049F6443-8E53-E011-A943-003048F117EA.root?tried=

      [h] Problematic data server: eoscms.cern.ch:1094

      [i] Disabled source: eoscms.cern.ch:1094

----- End Fatal Exception -------------------------------------------------

</pre>
140.53 step2
<pre>runTheMatrix-results/140.53_RunHI2011+RunHI2011+RECOHID11+HARVESTDHI/step2_RunHI2011+RunHI2011+RECOHID11+HARVESTDHI.log

----- Begin Fatal Exception 18-May-2015 14:30:26 UTC-----------------------

An exception of category 'FallbackFileOpenError' occurred while

   [0] Constructing the EventProcessor

   [1] Constructing input source of type PoolSource

   [2] Calling RootInputFileSequence::initFile()

   [3] Calling StorageFactory::open()

   [4] Calling XrdFile::open()

Exception Message:

Failed to open the file 'root://xrootd-cms.infn.it//store/hidata/HIRun2011/HIMinBiasUPC/RAW/v1/000/182/124/0011F55A-8F13-E111-A987-003048F118C6.root'

   Additional Info:

      [a] XrdCl::File::Open(name='root://eoscms//eos/cms/store/hidata/HIRun2011/HIMinBiasUPC/RAW/v1/000/182/124/0011F55A-8F13-E111-A987-003048F118C6.root?svcClass=default', flags=0x10, permissions=0660) => error '[ERROR] Server responded with an error: [3010] Unable to give access - user access restricted - not authorized identity used ; Permission denied

' (errno=3010, code=400). No additional data servers were found.

      [b] Last URL tried: root://eoscms:1094//eos/cms/store/hidata/HIRun2011/HIMinBiasUPC/RAW/v1/000/182/124/0011F55A-8F13-E111-A987-003048F118C6.root?svcClass=default&tried=

      [c] Problematic data server: eoscms:1094

      [d] Disabled source: eoscms:1094

      [e] Input file root://eoscms//eos/cms/store/hidata/HIRun2011/HIMinBiasUPC/RAW/v1/000/182/124/0011F55A-8F13-E111-A987-003048F118C6.root?svcClass=default could not be opened.

Fallback Input file root://xrootd-cms.infn.it//store/hidata/HIRun2011/HIMinBiasUPC/RAW/v1/000/182/124/0011F55A-8F13-E111-A987-003048F118C6.root also could not be opened.

Original exception info is above; fallback exception info is below.

      [f] XrdCl::File::Open(name='root://xrootd-cms.infn.it//store/hidata/HIRun2011/HIMinBiasUPC/RAW/v1/000/182/124/0011F55A-8F13-E111-A987-003048F118C6.root', flags=0x10, permissions=0660) => error '[ERROR] Server responded with an error: [3010] Unable to give access - user access restricted - not authorized identity used ; Permission denied

' (errno=3010, code=400). No additional data servers were found.

      [g] Last URL tried: root://eoscms.cern.ch:1094//store/hidata/HIRun2011/HIMinBiasUPC/RAW/v1/000/182/124/0011F55A-8F13-E111-A987-003048F118C6.root?tried=

      [h] Problematic data server: eoscms.cern.ch:1094

      [i] Disabled source: eoscms.cern.ch:1094

----- End Fatal Exception -------------------------------------------------

</pre>
1000.0 step2
<pre>runTheMatrix-results/1000.0_RunMinBias2011A+RunMinBias2011A+TIER0+SKIMD+HARVESTDfst2+ALCASPLIT/step2_RunMinBias2011A+RunMinBias2011A+TIER0+SKIMD+HARVESTDfst2+ALCASPLIT.log

----- Begin Fatal Exception 18-May-2015 14:31:54 UTC-----------------------

An exception of category 'FallbackFileOpenError' occurred while

   [0] Constructing the EventProcessor

   [1] Constructing input source of type PoolSource

   [2] Calling RootInputFileSequence::initFile()

   [3] Calling StorageFactory::open()

   [4] Calling XrdFile::open()

Exception Message:

Failed to open the file 'root://xrootd-cms.infn.it//store/data/Run2011A/MinimumBias/RAW/v1/000/165/121/0699429A-B37F-E011-A57A-0019B9F72D71.root'

   Additional Info:

      [a] XrdCl::File::Open(name='root://eoscms//eos/cms/store/data/Run2011A/MinimumBias/RAW/v1/000/165/121/0699429A-B37F-E011-A57A-0019B9F72D71.root?svcClass=default', flags=0x10, permissions=0660) => error '[ERROR] Server responded with an error: [3010] Unable to give access - user access restricted - not authorized identity used ; Permission denied

' (errno=3010, code=400). No additional data servers were found.

      [b] Last URL tried: root://eoscms:1094//eos/cms/store/data/Run2011A/MinimumBias/RAW/v1/000/165/121/0699429A-B37F-E011-A57A-0019B9F72D71.root?svcClass=default&tried=

      [c] Problematic data server: eoscms:1094

      [d] Disabled source: eoscms:1094

      [e] Input file root://eoscms//eos/cms/store/data/Run2011A/MinimumBias/RAW/v1/000/165/121/0699429A-B37F-E011-A57A-0019B9F72D71.root?svcClass=default could not be opened.

Fallback Input file root://xrootd-cms.infn.it//store/data/Run2011A/MinimumBias/RAW/v1/000/165/121/0699429A-B37F-E011-A57A-0019B9F72D71.root also could not be opened.

Original exception info is above; fallback exception info is below.

      [f] XrdCl::File::Open(name='root://xrootd-cms.infn.it//store/data/Run2011A/MinimumBias/RAW/v1/000/165/121/0699429A-B37F-E011-A57A-0019B9F72D71.root', flags=0x10, permissions=0660) => error '[ERROR] Server responded with an error: [3010] Unable to give access - user access restricted - not authorized identity used ; Permission denied

' (errno=3010, code=400). No additional data servers were found.

      [g] Last URL tried: root://eoscms.cern.ch:1094//store/data/Run2011A/MinimumBias/RAW/v1/000/165/121/0699429A-B37F-E011-A57A-0019B9F72D71.root?tried=

      [h] Problematic data server: eoscms.cern.ch:1094

      [i] Disabled source: eoscms.cern.ch:1094

----- End Fatal Exception -------------------------------------------------

</pre>
1001.0 step2
<pre>runTheMatrix-results/1001.0_RunMinBias2011A+RunMinBias2011A+TIER0EXP+ALCAEXP+ALCAHARVD/step2_RunMinBias2011A+RunMinBias2011A+TIER0EXP+ALCAEXP+ALCAHARVD.log

----- Begin Fatal Exception 18-May-2015 14:32:17 UTC-----------------------

An exception of category 'FallbackFileOpenError' occurred while

   [0] Constructing the EventProcessor

   [1] Constructing input source of type PoolSource

   [2] Calling RootInputFileSequence::initFile()

   [3] Calling StorageFactory::open()

   [4] Calling XrdFile::open()

Exception Message:

Failed to open the file 'root://xrootd-cms.infn.it//store/data/Run2011A/MinimumBias/RAW/v1/000/165/121/0699429A-B37F-E011-A57A-0019B9F72D71.root'

   Additional Info:

      [a] XrdCl::File::Open(name='root://eoscms//eos/cms/store/data/Run2011A/MinimumBias/RAW/v1/000/165/121/0699429A-B37F-E011-A57A-0019B9F72D71.root?svcClass=default', flags=0x10, permissions=0660) => error '[ERROR] Server responded with an error: [3010] Unable to give access - user access restricted - not authorized identity used ; Permission denied

' (errno=3010, code=400). No additional data servers were found.

      [b] Last URL tried: root://eoscms:1094//eos/cms/store/data/Run2011A/MinimumBias/RAW/v1/000/165/121/0699429A-B37F-E011-A57A-0019B9F72D71.root?svcClass=default&tried=

      [c] Problematic data server: eoscms:1094

      [d] Disabled source: eoscms:1094

      [e] Input file root://eoscms//eos/cms/store/data/Run2011A/MinimumBias/RAW/v1/000/165/121/0699429A-B37F-E011-A57A-0019B9F72D71.root?svcClass=default could not be opened.

Fallback Input file root://xrootd-cms.infn.it//store/data/Run2011A/MinimumBias/RAW/v1/000/165/121/0699429A-B37F-E011-A57A-0019B9F72D71.root also could not be opened.

Original exception info is above; fallback exception info is below.

      [f] XrdCl::File::Open(name='root://xrootd-cms.infn.it//store/data/Run2011A/MinimumBias/RAW/v1/000/165/121/0699429A-B37F-E011-A57A-0019B9F72D71.root', flags=0x10, permissions=0660) => error '[ERROR] Server responded with an error: [3010] Unable to give access - user access restricted - not authorized identity used ; Permission denied

' (errno=3010, code=400). No additional data servers were found.

      [g] Last URL tried: root://eoscms.cern.ch:1094//store/data/Run2011A/MinimumBias/RAW/v1/000/165/121/0699429A-B37F-E011-A57A-0019B9F72D71.root?tried=

      [h] Problematic data server: eoscms.cern.ch:1094

      [i] Disabled source: eoscms.cern.ch:1094

----- End Fatal Exception -------------------------------------------------

</pre>
1003.0 step2
<pre>runTheMatrix-results/1003.0_RunMinBias2012A+RunMinBias2012A+RECODDQM+HARVESTDDQM/step2_RunMinBias2012A+RunMinBias2012A+RECODDQM+HARVESTDDQM.log

----- Begin Fatal Exception 18-May-2015 14:33:33 UTC-----------------------

An exception of category 'FallbackFileOpenError' occurred while

   [0] Constructing the EventProcessor

   [1] Constructing input source of type PoolSource

   [2] Calling RootInputFileSequence::initFile()

   [3] Calling StorageFactory::open()

   [4] Calling XrdFile::open()

Exception Message:

Failed to open the file 'root://xrootd-cms.infn.it//store/data/Run2012A/MinimumBias/RAW/v1/000/191/226/02F7EB61-0986-E111-A66F-5404A63886B4.root'

   Additional Info:

      [a] XrdCl::File::Open(name='root://eoscms//eos/cms/store/data/Run2012A/MinimumBias/RAW/v1/000/191/226/02F7EB61-0986-E111-A66F-5404A63886B4.root?svcClass=default', flags=0x10, permissions=0660) => error '[ERROR] Server responded with an error: [3010] Unable to give access - user access restricted - not authorized identity used ; Permission denied

' (errno=3010, code=400). No additional data servers were found.

      [b] Last URL tried: root://eoscms:1094//eos/cms/store/data/Run2012A/MinimumBias/RAW/v1/000/191/226/02F7EB61-0986-E111-A66F-5404A63886B4.root?svcClass=default&tried=

      [c] Problematic data server: eoscms:1094

      [d] Disabled source: eoscms:1094

      [e] Input file root://eoscms//eos/cms/store/data/Run2012A/MinimumBias/RAW/v1/000/191/226/02F7EB61-0986-E111-A66F-5404A63886B4.root?svcClass=default could not be opened.

Fallback Input file root://xrootd-cms.infn.it//store/data/Run2012A/MinimumBias/RAW/v1/000/191/226/02F7EB61-0986-E111-A66F-5404A63886B4.root also could not be opened.

Original exception info is above; fallback exception info is below.

      [f] XrdCl::File::Open(name='root://xrootd-cms.infn.it//store/data/Run2012A/MinimumBias/RAW/v1/000/191/226/02F7EB61-0986-E111-A66F-5404A63886B4.root', flags=0x10, permissions=0660) => error '[ERROR] Server responded with an error: [3010] Unable to give access - user access restricted - not authorized identity used ; Permission denied

' (errno=3010, code=400). No additional data servers were found.

      [g] Last URL tried: root://eoscms.cern.ch:1094//store/data/Run2012A/MinimumBias/RAW/v1/000/191/226/02F7EB61-0986-E111-A66F-5404A63886B4.root?tried=

      [h] Problematic data server: eoscms.cern.ch:1094

      [i] Disabled source: eoscms.cern.ch:1094

----- End Fatal Exception -------------------------------------------------

</pre>
25202.0 step2
<pre>runTheMatrix-results/25202.0_TTbar_13+TTbar_13+DIGIUP15_PU25+RECOUP15_PU25+HARVESTUP15_PU25+MINIAODMCUP15/step2_TTbar_13+TTbar_13+DIGIUP15_PU25+RECOUP15_PU25+HARVESTUP15_PU25+MINIAODMCUP15.log

----- Begin Fatal Exception 18-May-2015 14:43:39 UTC-----------------------

An exception of category 'FallbackFileOpenError' occurred while

   [0] Constructing the EventProcessor

   [1] Constructing module: class=MixingModule label='mix'

   [2] Calling RootInputFileSequence::initFile()

   [3] Calling StorageFactory::open()

   [4] Calling XrdFile::open()

Exception Message:

Failed to open the file 'root://xrootd-cms.infn.it//store/relval/CMSSW_7_5_0_pre4/RelValMinBias_13/GEN-SIM/MCRUN2_75_V1-v1/00000/3E2E2F78-BFF5-E411-94EE-0025905A60D6.root'

   Additional Info:

      [a] XrdCl::File::Open(name='root://eoscms//eos/cms/store/relval/CMSSW_7_5_0_pre4/RelValMinBias_13/GEN-SIM/MCRUN2_75_V1-v1/00000/3E2E2F78-BFF5-E411-94EE-0025905A60D6.root?svcClass=default', flags=0x10, permissions=0660) => error '[ERROR] Server responded with an error: [3010] Unable to give access - user access restricted - not authorized identity used ; Permission denied

' (errno=3010, code=400). No additional data servers were found.

      [b] Last URL tried: root://eoscms:1094//eos/cms/store/relval/CMSSW_7_5_0_pre4/RelValMinBias_13/GEN-SIM/MCRUN2_75_V1-v1/00000/3E2E2F78-BFF5-E411-94EE-0025905A60D6.root?svcClass=default&tried=

      [c] Problematic data server: eoscms:1094

      [d] Disabled source: eoscms:1094

      [e] Input file root://eoscms//eos/cms/store/relval/CMSSW_7_5_0_pre4/RelValMinBias_13/GEN-SIM/MCRUN2_75_V1-v1/00000/3E2E2F78-BFF5-E411-94EE-0025905A60D6.root?svcClass=default could not be opened.

Fallback Input file root://xrootd-cms.infn.it//store/relval/CMSSW_7_5_0_pre4/RelValMinBias_13/GEN-SIM/MCRUN2_75_V1-v1/00000/3E2E2F78-BFF5-E411-94EE-0025905A60D6.root also could not be opened.

Original exception info is above; fallback exception info is below.

      [f] XrdCl::File::Open(name='root://xrootd-cms.infn.it//store/relval/CMSSW_7_5_0_pre4/RelValMinBias_13/GEN-SIM/MCRUN2_75_V1-v1/00000/3E2E2F78-BFF5-E411-94EE-0025905A60D6.root', flags=0x10, permissions=0660) => error '[ERROR] Server responded with an error: [3010] Unable to give access - user access restricted - not authorized identity used ; Permission denied

' (errno=3010, code=400). No additional data servers were found.

      [g] Last URL tried: root://eoscms.cern.ch:1094//store/relval/CMSSW_7_5_0_pre4/RelValMinBias_13/GEN-SIM/MCRUN2_75_V1-v1/00000/3E2E2F78-BFF5-E411-94EE-0025905A60D6.root?tried=

      [h] Problematic data server: eoscms.cern.ch:1094

      [i] Disabled source: eoscms.cern.ch:1094

----- End Fatal Exception -------------------------------------------------

</pre>
50202.0 step2
<pre>runTheMatrix-results/50202.0_TTbar_13+TTbar_13+DIGIUP15_PU50+RECOUP15_PU50+HARVESTUP15_PU50+MINIAODMCUP1550/step2_TTbar_13+TTbar_13+DIGIUP15_PU50+RECOUP15_PU50+HARVESTUP15_PU50+MINIAODMCUP1550.log

----- Begin Fatal Exception 18-May-2015 14:43:47 UTC-----------------------

An exception of category 'FallbackFileOpenError' occurred while

   [0] Constructing the EventProcessor

   [1] Constructing module: class=MixingModule label='mix'

   [2] Calling RootInputFileSequence::initFile()

   [3] Calling StorageFactory::open()

   [4] Calling XrdFile::open()

Exception Message:

Failed to open the file 'root://xrootd-cms.infn.it//store/relval/CMSSW_7_5_0_pre4/RelValMinBias_13/GEN-SIM/MCRUN2_75_V1-v1/00000/5E626486-C3F5-E411-BFF6-0025905B85EE.root'

   Additional Info:

      [a] XrdCl::File::Open(name='root://eoscms//eos/cms/store/relval/CMSSW_7_5_0_pre4/RelValMinBias_13/GEN-SIM/MCRUN2_75_V1-v1/00000/5E626486-C3F5-E411-BFF6-0025905B85EE.root?svcClass=default', flags=0x10, permissions=0660) => error '[ERROR] Server responded with an error: [3010] Unable to give access - user access restricted - not authorized identity used ; Permission denied

' (errno=3010, code=400). No additional data servers were found.

      [b] Last URL tried: root://eoscms:1094//eos/cms/store/relval/CMSSW_7_5_0_pre4/RelValMinBias_13/GEN-SIM/MCRUN2_75_V1-v1/00000/5E626486-C3F5-E411-BFF6-0025905B85EE.root?svcClass=default&tried=

      [c] Problematic data server: eoscms:1094

      [d] Disabled source: eoscms:1094

      [e] Input file root://eoscms//eos/cms/store/relval/CMSSW_7_5_0_pre4/RelValMinBias_13/GEN-SIM/MCRUN2_75_V1-v1/00000/5E626486-C3F5-E411-BFF6-0025905B85EE.root?svcClass=default could not be opened.

Fallback Input file root://xrootd-cms.infn.it//store/relval/CMSSW_7_5_0_pre4/RelValMinBias_13/GEN-SIM/MCRUN2_75_V1-v1/00000/5E626486-C3F5-E411-BFF6-0025905B85EE.root also could not be opened.

Original exception info is above; fallback exception info is below.

      [f] XrdCl::File::Open(name='root://xrootd-cms.infn.it//store/relval/CMSSW_7_5_0_pre4/RelValMinBias_13/GEN-SIM/MCRUN2_75_V1-v1/00000/5E626486-C3F5-E411-BFF6-0025905B85EE.root', flags=0x10, permissions=0660) => error '[ERROR] Server responded with an error: [3010] Unable to give access - user access restricted - not authorized identity used ; Permission denied

' (errno=3010, code=400). No additional data servers were found.

      [g] Last URL tried: root://eoscms.cern.ch:1094//store/relval/CMSSW_7_5_0_pre4/RelValMinBias_13/GEN-SIM/MCRUN2_75_V1-v1/00000/5E626486-C3F5-E411-BFF6-0025905B85EE.root?tried=

      [h] Problematic data server: eoscms.cern.ch:1094

      [i] Disabled source: eoscms.cern.ch:1094

----- End Fatal Exception -------------------------------------------------

</pre>

 you can see the results of the tests here: 
 https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-9127/81/summary.html

--- a/HLTriggerOffline/Higgs/python/hltHiggsValidator_cfi.py
+++ b/HLTriggerOffline/Higgs/python/hltHiggsValidator_cfi.py
@@ -60,14 +60,13 @@ hltHiggsValidator = cms.EDAnalyzer("HLTHiggsValidator",
     # --- Photons
     Photon_genCut     = cms.string("abs(pdgId) == 22 && status == 1"),
     Photon_recCut     = cms.string("pt > 20 && abs(eta) < 2.4 && hadronicOverEm < 0.1 && "+\
-                                       " ( abs(eta) < 1.479 && r9 < 0.50  || "+\
-                                       "   abs(eta) > 1.479 && r9 < 0.80 ) && "+\
-                                       " ( ( abs(eta) < 1.479 && r9 < 0.85  || "+\
-                                       "   abs(eta) > 1.479 && r9 < 0.90 ) || ("+\
-                                       " ( abs(eta) < 1.479 && sigmaIetaIeta < 0.014  || "+\
-                                       "   abs(eta) > 1.479 && sigmaIetaIeta < 0.0035 ) && "+\
-                                       " ecalRecHitSumEtConeDR03 < (6.0+0.012*et)  && trkSumPtSolidConeDR03 < (6.0 + 0.0002*et)"+\
-            " )"+")" ),
+                                   " ( ( abs(eta) < 1.479 && r9 > 0.85 ) || "+\
+                                   "   ( abs(eta) > 1.479 && r9 > 0.90 ) || "+\
+                                   "   ( abs(eta) < 1.479 && r9 > 0.50 && sigmaIetaIeta < 0.014 && "+\
+                                   "     ecalRecHitSumEtConeDR03 < (6.0+0.012*et) && trkSumPtSolidConeDR03 < (6.0 + 0.002*et) ) || "+\
+                                   "   ( abs(eta) > 1.479 && r9 > 0.80 && sigmaIetaIeta < 0.035 && "+\
+                                   "     ecalRecHitSumEtConeDR03 < (6.0+0.012*et) && trkSumPtSolidConeDR03 < (6.0 + 0.002*et) ) ) "
+                                   ),
     Photon_cutMinPt   = cms.double(20), # TO BE DEPRECATED
     Photon_cutMaxEta  = cms.double(2.4),# TO BE DEPRECATED
 

--- a/HLTriggerOffline/Higgs/python/hltHiggsValidator_cfi.py
+++ b/HLTriggerOffline/Higgs/python/hltHiggsValidator_cfi.py
@@ -59,10 +59,14 @@ hltHiggsValidator = cms.EDAnalyzer("HLTHiggsValidator",
 
     # --- Photons
     Photon_genCut     = cms.string("abs(pdgId) == 22 && status == 1"),
-    Photon_recCut     = cms.string("pt > 20 && abs(eta) < 2.4 && hadronicOverEm < 0.1 && ( r9 < 0.85 || ("+\
-            " ( abs(eta) < 1.479 && sigmaIetaIeta < 0.014  || "+\
-            "   abs(eta) > 1.479 && sigmaIetaIeta < 0.0035 ) && "+\
-            " ecalRecHitSumEtConeDR03 < (5.0+0.012*et) && hcalTowerSumEtConeDR03 < (5.0+0.0005*et )  && trkSumPtSolidConeDR03 < (5.0 + 0.0002*et)"+\
+    Photon_recCut     = cms.string("pt > 20 && abs(eta) < 2.4 && hadronicOverEm < 0.1 && "+\
+                                       " ( abs(eta) < 1.479 && r9 < 0.50  || "+\
+                                       "   abs(eta) > 1.479 && r9 < 0.80 ) && "+\
+                                       " ( ( abs(eta) < 1.479 && r9 < 0.85  || "+\
+                                       "   abs(eta) > 1.479 && r9 < 0.90 ) || ("+\
+                                       " ( abs(eta) < 1.479 && sigmaIetaIeta < 0.014  || "+\
+                                       "   abs(eta) > 1.479 && sigmaIetaIeta < 0.0035 ) && "+\
+                                       " ecalRecHitSumEtConeDR03 < (6.0+0.012*et)  && trkSumPtSolidConeDR03 < (6.0 + 0.0002*et)"+\
             " )"+")" ),
     Photon_cutMinPt   = cms.double(20), # TO BE DEPRECATED
     Photon_cutMaxEta  = cms.double(2.4),# TO BE DEPRECATED
@@ -151,12 +155,12 @@ hltHiggsValidator = cms.EDAnalyzer("HLTHiggsValidator",
         ),
     Hgg = cms.PSet( 
         hltPathsToCheck = cms.vstring(
-            "HLT_Diphoton44_28_R9Id85_OR_Iso50T80LCaloId24b40e_AND_HE10P1_R9Id50b80e_v",
-            "HLT_Diphoton30_18_R9Id85_OR_Iso50T80LCaloId24b40e_AND_HE10P0_R9Id50b80e_Mass95_v",
-            "HLT_Diphoton28_14_R9Id85_OR_Iso50T80LCaloId24b40e_AND_HE10P0_R9Id50b80e_Mass50_Eta_1p5_v",
-            "HLT_Diphoton30_18_R9Id85_AND_Iso50T80LCaloId24b40e_AND_HE10P0_R9Id50b80e_Solid_Mass30_v",
-            "HLT_Diphoton30_18_R9Id85_AND_Iso50T80LCaloId24b40e_AND_HE10P0_R9Id50b80e_PV_v",
-            "HLT_Diphoton30_18_R9Id85_AND_Iso50T80LCaloId24b40e_AND_HE10P0_R9Id50b80e_DoublePV_v"
+            "HLT_Diphoton10_10_R9Id_OR_IsoCaloId_AND_HE_R9Id_Mass10_ForMC_v",
+            "HLT_Diphoton30_18_R9Id_OR_IsoCaloId_AND_HE_R9Id_Mass95_v",
+            "HLT_Diphoton30_18_R9Id_OR_IsoCaloId_AND_HE_R9Id_DoublePixelSeedMatch_Mass70_v",
+            "HLT_Diphoton30PV_18PV_R9Id_AND_IsoCaloId_AND_HE_R9Id_DoublePixelVeto_Mass55_v",
+            "HLT_Diphoton30_18_Solid_R9Id_AND_IsoCaloId_AND_HE_R9Id_Mass55_v",
+            "HLT_Diphoton30EB_18EB_R9Id_OR_IsoCaloId_AND_HE_R9Id_DoublePixelVeto_Mass55_v"
         ),
         recPhotonLabel  = cms.string("photons"),
         # -- Analysis specific cuts

